src/Core/AspectKernel.php

```diff
@@ -254,7 +284,7 @@ protected function registerTransformers()
                 $transformers[] = new ConstructorExecutionTransformer();
             }
 
-            return $transformers;
+            return $aspectKernel->customizeTransformers($transformers);
```

Comment:
Method `customizeTransformers()` will be removed because it's no needed. Just override this method itself in your application to customize the list of extensions.